README.md

```diff
@@ -1,36 +1,159 @@
 
-A Python wrapper for the [MX Atrium API](https://atrium.mx.com). In order to make requests, you will need to [sign up for the MX Atrium API](https://atrium.mx.com/developers/sign_up) and get a `MX-API-KEY` and a  `MX-CLIENT-ID`.
+## Requirements.
 
-### Usage
+Python 2.7 and 3.4+
 
-Add the `atrium` module to your source code's directory and import into your source code file with the following
-```python
-from atriumclient import AtriumClient
+## Installation & Usage
+### pip install
+
+```sh
+pip install git+https://github.com/mxenabled/atrium-python.git
 ```
+(you may need to run `pip` with root permission: `sudo pip install git+https://github.com/mxenabled/atrium-python.git`)
 
-Then configure your instance with the following. (The `ENVIRONMENT` will be either `vestibule.mx.com` for the development environment or `atrium.mx.com` for the production environment.)
+Then import the package:
 ```python
-atriumClient = AtriumClient("ENVIRONMENT", "YOUR_MX_API_KEY", "YOUR_MX_CLIENT_ID")
+import atrium
 ```
 
-Then start using class methods to make calls to the Atrium API for data. See the full [Atrium documentation](https://atrium.mx.com/documentation) for more details.
+### Setuptools
 
-```python
-# import AtriumClient wrapper class
-from atriumclient import AtriumClient
+Install via [Setuptools](http://pypi.python.org/pypi/setuptools).
 
-# Configure AtriumClient
-atriumClient = AtriumClient("ENVIRONMENT", "YOUR_MX_API_KEY", "YOUR_MX_CLIENT_ID")
+```sh
+python setup.py install --user
+```
+(or `sudo python setup.py install` to install the package for all users)
 
-# Now begin making Atrium calls
-atriumClient.createUser(identifier = "UniqueID") # Create a user, etc...
+Then import the package:
+```python
+import atrium
 ```
 
+## Example Usage
 
-### Examples
+Please see `docs` directory for additional endpoint examples
 
-The `/examples` directory contains various workflows and code snippets. You will first need to modify the line shown below in each example with the environment, YOUR-MX-API-KEY, and YOUR-MX-CLIENT-ID before running.
 ```python
-atriumClient = AtriumClient("ENVIRONMENT", "YOUR_MX_API_KEY", "YOUR_MX_CLIENT_ID")
+from __future__ import print_function
+import time
+import atrium
+from atrium.rest import ApiException
+from pprint import pprint
+
+# Configure API Key authorization
+configuration = atrium.Configuration()
+configuration.api_key['MX-API-Key'] = 'YOUR_API_KEY'
+
+# Configure Client ID authorization
+configuration = atrium.Configuration()
+configuration.api_key['MX-Client-ID'] = 'YOUR_CLIENT_ID'
```

Comment:
This is a bit confusing. `api_key` is a dictionary with different ids?

Reply:
Yes, I agree. Swagger uses `api_key` for the dictionary name, which is confusing when our keys are named `API_Key` and `Client_ID`. 

I'm thinking about renaming the dictionary to `credentials`, or maybe `auth`. Do you think that would help clear it up?

Reply:
Hmm. I am not sure, what else is on the configuration object?

Reply:
Some url, auth, ssl/tls, and proxy settings. [Here](https://github.com/mxenabled/atrium-python/blob/2cc37851a55630ae27d83be71d909ed0aff99d1c/atrium/configuration.py#L37-L92) are the specific fields.

I also think it could be cleared up by simply making it plural, e.g. `api_keys`

Reply:
Maybe it would be better to call these headers? That is what they are at the end of the day aren't they? But I could be missing out on a ton of context. Is this the default name that Swagger provides or something?